main.tf

```diff
@@ -43,7 +64,17 @@ resource "aws_ssm_parameter" "this" {
   name  = "${local.name}-secret"
   type  = "String"
   value = var.pem
-  tier  = "Advanced" # value is over 4kb
+  tier  = "Intelligent-Tiering"
+  tags = local.tags
+}
+
+resource "aws_ssm_parameter" "ca_crt" {
```

Comment:
ca.crt does not contain any security-sensitive data like private keys -> there is no need to use ssm for it

Reply:
not part of my changes but can be removed as part of this PR